src/workerd/api/queue.h

```diff
@@ -30,11 +30,10 @@ class WorkerQueue: public jsg::Object {
   struct SendOptions {
     // TODO(soon): Support metadata.
 
-    jsg::Optional<bool> tmp;
-    // TODO(soon): Remove this -- it's only here to make JSG_STRUCT work since it doesn't if a
-    // struct doesn't have any fields.
+    jsg::Optional<kj::String> contentType;
+    // contentType determines the serialization format of the message.
 
-    JSG_STRUCT(tmp);
+    JSG_STRUCT(contentType);
     JSG_STRUCT_TS_OVERRIDE(QueueSendOptions {
       tmp: never;
```

Comment:
We should just be able to remove this line now that `tmp` isn't defined in the struct, right?

Reply:
I wasn't 100% sure about that. The struct does has a real field now, so we don't need `tmp` anymore.
But on the other hand, I'd rather hold off on exposing this new field in workers-types for the moment (until I've let it roll out and tested it).
Know the right way to achieve that? I currently went with the approach of just leaving the override unchanged.

Reply:
If you don't want `contentType` to show up, then I think you'd want to replace this with `contentType: never`?

If my bash history is anything to go by, you should be able to verify what output this generates by running `bazel build //types:types` and then checking the contents of `bazel-bin/types/definitions/experimental/index.d.ts`